actions/templates/install/index.js

```diff
@@ -171,10 +181,16 @@ async function main (params) {
         if (apiFlowType.toLowerCase() !== CREDENTIAL_FLOW_TYPE_ADOBEID || apiCredentialType.toLowerCase() !== credentialType.toLowerCase()) {
           continue;
         }
+        // extract api license config from the api info map
+        let apiLicenseConfigs = [];
+        const apiInfo = mapApiCodeToApiInfo[api.code];
+        if (apiInfo && apiInfo.credentialType.toLowerCase() === apiCredentialType.toLowerCase() && apiInfo.flowType.toLowerCase() === apiFlowType.toLowerCase()) {
+          apiLicenseConfigs = apiInfo.licenseConfigs;
+        }
```

Comment:
Duplicate code below

Reply:
For `adobeid` type credentials, it would be nice to clarify in a comment that they do not support license configs because they do not have a technical account. However, we keep this logic here because TR doesn't need to know this distinction between credential types and console will simply ignore in case license configs are included.

Reply:
Okay sure. I wasn't aware that `adobeid` type credentials do not support license configs. Thanks for letting know, I'll place a comment there.